Hangman/Hangman.py

```diff
@@ -0,0 +1,53 @@
+class Hangman:
```

Comment:
we can better structure this project. 
folders like models, constants, tests etc.